<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>606</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>606</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Academy Engraved LET</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="input_text_label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>1</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Chalkduster</family>
      <pointsize>26</pointsize>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Enter an expression:</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::LinksAccessibleByMouse</set>
    </property>
   </widget>
   <widget class="QLabel" name="result_label">
    <property name="geometry">
     <rect>
      <x>337</x>
      <y>240</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkduster</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Your result:</string>
    </property>
   </widget>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>337</x>
      <y>270</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkduster</family>
      <pointsize>24</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>4</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>511</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkduster</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="result_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>67</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkduster</family>
      <pointsize>31</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="input_x_label">
    <property name="geometry">
     <rect>
      <x>337</x>
      <y>100</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkduster</family>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X =</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="graphic" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>321</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Wingdings 2</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x_min_box">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>350</y>
      <width>148</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkduster</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="accelerated">
     <bool>false</bool>
    </property>
    <property name="correctionMode">
     <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
    </property>
    <property name="keyboardTracking">
     <bool>true</bool>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
    <property name="value">
     <double>-15.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="x_max_box">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>350</y>
      <width>148</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkduster</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="value">
     <double>15.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="y_min_box">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>350</y>
      <width>148</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkduster</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="value">
     <double>-15.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="y_max_box">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>350</y>
      <width>148</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkduster</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="value">
     <double>15.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="input_text_x">
    <property name="geometry">
     <rect>
      <x>393</x>
      <y>100</y>
      <width>204</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkduster</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>37</x>
      <y>323</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkduster</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X Min</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>188</x>
      <y>323</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkduster</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X Max</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>323</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkduster</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y Min</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>323</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chalkduster</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y Max</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSmartCalc">
    <property name="title">
     <string>SmartCalc</string>
    </property>
   </widget>
   <addaction name="menuSmartCalc"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
